config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml

```diff
@@ -44,6 +44,11 @@ periodics:
   spec:
     containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20191001-7ca508a-master
+        env:
```

Comment:
@detiber does `master` work here? based on the change you suggested in https://github.com/kubernetes/kubernetes/issues/81760 ?

Reply:
I believe it should :)